df-valve/src/test/java/com/hashmapinc/tempus/witsml/valve/dot/DotValveTest.java

```diff
@@ -90,6 +90,32 @@ public void createObjectWell1411() throws IOException, JAXBException{
         assertNotNull(uid);
```

Comment:
I added a 1311 and 1411 well XML query file to test resources

I added a test for each WELL version for getting by UID. The UID is currently based on the dummy data that `Drilltest` autopopulates.

The tests simply ensure that any data is returned. More advanced testing is needed to actually check the quality of the data returned.